cross/dfu-programmer/Portfile

```diff
@@ -1,26 +1,28 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
+PortSystem          1.0
 
-name              dfu-programmer
-version           0.6.2
-categories        cross devel
-maintainers       vogelgsang.org:chris
-description       dfu-programmer is a Device Firmware Update based USB programmer for Atmel chips with a USB bootloader.
-long_description  dfu-programmer is an implementation of the Device Firmware Upgrade class \
-                  USB driver that enables firmware upgrades for various USB enabled (with the \
-                  correct bootloader) Atmel chips.  This program was created because the \
-                  Atmel "FLIP" program for flashing devices does not support flashing via USB \
-                  on Linux, and because standard DFU loaders do not work for Atmel's chips.
-homepage          http://dfu-programmer.sourceforge.net/
-platforms         darwin
-master_sites      sourceforge
+name                dfu-programmer
+version             0.7.2
+categories          cross devel
+license             GPL-2
+maintainers         nomaintainer
```

Comment:
Does the maintainer agree to this? Was a port abandoned ticket filed?

Reply:
was not updated by anyone including the maintainer for the last ~6 years; I remember a policy like this in the past. Am I wrong?

Reply:
The policy is that a port abandoned ticket must be filed before a maintainer can be removed. 

See [section 7.4.2](https://guide.macports.org/chunked/project.update-policies.html) of the guide.

Reply:
Sorry for the mistake. Ticket [created](https://trac.macports.org/ticket/59849). Now we wait 72 hours.
Thank you

Reply:
changed commit with `Closes` line